Dockerfile

```diff
@@ -38,4 +38,4 @@ RUN python3 setup.py install
 
 STOPSIGNAL SIGINT
 
-ENTRYPOINT [ "python3", "-m", "gprofiler", "-v" ]
```

Comment:
The FileHandler still takes all DEBUG logs, so I guess that's alright.

Reply:
I think we can leave the FileHandler on DEBUG level, and leave the option to user to decide on verbose log to stdout.
Right now the verbose is "hardcoded" and there is not option in `docker` to see only logs > INFO

Reply:
kk